pkg/pgmodel/ingestor/metrics.go

```diff
@@ -89,6 +84,13 @@ var (
 		},
 		func() float64 { return float64(len(CopierChannelToMonitor)) },
 	)
+	currentIngestions = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Namespace: util.PromNamespace,
+			Name:      "current_ingestions",
```

Comment:
should this be write_requests_processing to be more understandable? And should this then be measured from write.go? Should we have both?

Reply:
> should this then be measured from write.go

active write requests are part of ingestor. We can declare it here and use it in write.go for better accuracy.

> Should we have both?

Do you mean, one metric over here and another metric in api/write.go? I feel that becomes trivial since that's not our aim here. WDYT?